src/frontend/src/components/Post/Post.jsx

```diff
@@ -6,6 +6,7 @@ import { makeStyles } from '@material-ui/core/styles';
 import { Box, Grid, Typography, ListSubheader } from '@material-ui/core';
 import './telescope-post-content.css';
 import AdminButtons from '../AdminButtons';
+import { autoDetection } from 'highlight.js';
```

Comment:
Is it just me? I don't see this being used anywhere in the code change.

Reply:
we have this:
```js
import 'highlight.js/styles/github.css';
```
in line 4. So what is this in line 9 for?

Reply:
oh shoot. I think I was testing a fix for the overflown content mentioned above. I will remove it. I apologize.

Reply:
Good catch @c3ho I will check for that in the future. Sorry about that again.